getting-started/vs-integration/upgrade-wizard.md

```diff
@@ -35,47 +33,28 @@ The next page of the Upgrade Wizard lists all projects from the solution that us
 
-The option is selected by default if there is a **Resources\TelerikMessages.resx** file in the current project. In this case, **Resources\TelerikMessages.resx** file is selected in the file browser.
-
-![Upgrade Wizard Localization File](images/upgrade-wizard-localization-file.png)  
+The option is selected by default if there is a **Resources\TelerikMessages.resx** file in the current project. In this case, **Resources\TelerikMessages.resx** file is selected in the file browser. 
 
 ## Create Backup
 
 You can create a backup before running the upgrade wizard by selecting the **Create a backup before upgrading** option.
 
 >When the project and solution are placed in the same directory, the default backup location is set to the **%USERPROFILE%\Documents** folder.
 
-![Upgrade Wizard Backup](images/upgrade-wizard-backup.png)  
-
-
 ## Upgrade Log
 
-In case one of the options ([Create a backup before upgrading](#create-backup) or [Upgrade the localization file](#upgrade-localization-file)) is selected, an **Upgrade Log** will be opened in your default browser at the end of the upgrade. The log can show the backup location, an entry for each localization key added or a localization upgrade error. Each upgrade log file is in HTML format. The file name contains the date and time of the upgrade.
-
-![Upgrade Wizard Log](images/upgrade-wizard-backup.png)
+In case one of the options ([Create a backup before upgrading](#create-backup) or [Upgrade the localization file](#upgrade-localization-file)) is selected, an **Upgrade Log** will be opened at the end of the upgrade. The log can show the backup location, an entry for each localization key added or a localization upgrade error. Each upgrade log file is in HTML format. The file name contains the date and time of the upgrade.
 
 ## Troubleshooting
 
-The following common problems might occur when upgrading a project:
-
-* The [localization file is read-only or does not exists](#missing-localization-file).
-* The [**Finish** button is disabled](#finish-button-is-disabled).
-
-### Missing Localization file
-
-If the selected localization file in the pointed directory is missing or read-only, the page state is not valid. In this case, the **Finish** button will display a tooltip when clicked, indicating a page error.
-
-![Upgrade Wizard Localization File](images/upgrade-wizard-localization-file-invalid.png)
-
-### Finish Button is Disabled
+The **Finish** button will display a tooltip, indicating a page error when:
 
-If **Create a backup before upgrading** option is selected, and the **Backup location** is empty, the **Finish** button is disabled.
+* The **Upgrade the localization file** option is selected and the pointed **directory is empty** or the file **is read-only**.
```

Comment:
We shouldn't use bold for emphasizing. There are places we have done that in the past, but it's not compliant with the Style Guide.